### DISCUSSION TITLE
When will new board come up in Arduino IDE?

### DISCUSSION THREAD

**Discussion by SooDragon** [espressif/arduino-esp32#9310] [Q&A] [answered]
I had committed new ESP32-C3 custom board and it's merged few days ago.
https://github.com/espressif/arduino-esp32/pull/9260

and I wonder when can we find this board in Arduino IDE.

![Screenshot 2024-02-29 at 20 57 09](https://github.com/espressif/arduino-esp32/assets/82627949/ceb282b8-55a1-40f3-8aac-450eb8b17b86)

1. When can we find new boards in Arduino IDE?
2. Which version will the board belong to?

**Comment by P-R-O-C-H-Y** [comment, marked as answer]
Hi @SooDragon, you won't be able to find the board in old releases. Your board will be included in the next release of the ESP32 Arduino core. Now you have 2 options.
Option 1 is to wait until next release, which will include your board (it will be release 3.0.0-rc1)
Option 2 is to use manual installation by cloning the git repository. If you use the base branch(master) your board will show up as its already merged :) You can check the docs for manual installation -> [Link](https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html)

**Reply by SooDragon** [reply]
Thank you for answer!
 I tried to find the expected release date for 3.0.0-rc1, but couldn't find it. Could you please ask where I can find it?

**Reply by P-R-O-C-H-Y** [reply]
There is no official date for 3.0.0-rc1 release, but we are doing everything to release as soon as possible. The release should happen within few weeks, once the Network refactoring is finished.

**Reply by SooDragon** [reply]
Thank you!
